### PR TITLE
Fix live test image when running in CI

### PR DESCRIPTION
**Pull request by clnoll**
## What
Fixes an issue where regression tests were using the same target & control containers when running in GHA.

This was flagged by @aldogonzalez8 when he realized that the target & control containers were hitting the same URLs, when they were expected to be different.

## How
Avoid looking for a `connection_id.txt` file when in CI and getting the control container. When this file is present, we were building the control image off of it instead of pulling the latest image from Dockerhub, however it should only be used for the target image.

Closes https://github.com/airbytehq/airbyte-internal-issues/issues/7950.

### CODE REVIEW

File: airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py [hunk 1]
```diff
@@ -84,22 +84,25 @@ async def get_container_from_dockerhub_image(dagger_client: dagger.Client, docke
...
         dagger.Container: The dagger container for the connector image to test
     """
     # If a container_id.txt file is available, we'll use it to load the connector container
     # We use a txt file as container ids can be too long to be passed as env vars
     # It's used for dagger-in-dagger use case with airbyte-ci, when the connector container is built via an upstream dagger operation
-    container_id_path = Path("/tmp/container_id.txt")
-    if container_id_path.exists():
-        return await get_container_from_id(dagger_client, container_id_path.read_text())
+    if is_ci and is_target:
+        container_id_path = Path("/tmp/container_id.txt")
```
Comment (alafanechere) [inline airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py:103]: In order to not tie this utils to the regression test suite I'd suggest to make the airbyte-ci `RegressionTests` step write `/tmp/<docker-image-name-with-tag>_container_id.txt` and make this function check if such a file is available.

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py (hunk 1 above)
Comment (clnoll) [reply airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py:103]: I have a vague memory of having done that in an earlier version of the code 😄. Will wire it through. But, it is a slightly bigger lift so would you be open to approving this PR so we can unblock people and I'll do it in a follow up?

Comment (clnoll) [reply airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py:103]: Actually on second thought I think what you're proposing is slightly lower effort than what I was envisioning. Will do it now.

Comment (alafanechere) [reply airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py:103]: Let me know if it's too much, happy to 👍  to unblock.

Comment (clnoll) [reply airbyte-ci/connectors/live-tests/src/live_tests/commons/utils.py]: Thanks @alafanechere I made this change.

**Review by alafanechere — APPROVED** [review]
@clnoll nice! Pre-approving! Can you double check it works? I wonder if it's risky to create file name with special characters like `:` (the image name with tag will contain a `:`).

**Review by bnchrch — APPROVED** [review]

**Comment by clnoll** [comment]
@alafanechere the ":" indeed did not work so I'm just using the target image tag instead. Sound okay to you?

**Comment by alafanechere** [comment]
> @alafanechere the ":" indeed did not work so I'm just using the target image tag instead. Sound okay to you?

I think you can use the `slugify` helper.

**Pull request merged**